multipaz/src/commonMain/kotlin/org/multipaz/sdjwt/Issuer.kt

```diff
@@ -13,11 +14,14 @@ import org.multipaz.crypto.Algorithm
  * @param kid a parameter further identifying the key, if necessary (e.g., when the
  *        iss URL points to a file with multiple keys. This parameter will be copied
  *        into the header of the JWT.
+ * @param x5c a certificate chain, from the signer of the JWT up to some authority that
+ *        the verifier of the JWT will likely trust. Optional.
  */
 data class Issuer(
     val iss: String,
     val alg: Algorithm,
-    val kid: String? = null /* optional */
+    val kid: String? = null /* optional */,
+    val x5c: X509CertChain? = null /* optional */
```

Comment:
No need to add the `/* optional */` comments, that's obvious from the fact it's nullable.

Reply:
Done.